CMakeLists.txt

```diff
@@ -3,41 +3,43 @@
 #                          Raphael Dumusc <raphael.dumusc@epfl.ch>
 #                          Daniel Nachbaur <daniel.nachbaur@epfl.ch>
 
-cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
-project(Deflect)
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
+project(Deflect VERSION 0.10.1)
 
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake
                               ${CMAKE_SOURCE_DIR}/CMake/common)
 include(GitExternal)
 
-set(VERSION_MAJOR 0)
-set(VERSION_MINOR 10)
-set(VERSION_PATCH 1)
-set(VERSION_ABI 3)
+set(Deflect_VERSION_ABI 3)
```

Comment:
OK I see, that's fine then. For cosmetics I would place this line right under project(Deflect VERSION 0.10.1)